docs/getting-started/install.mdx

```diff
@@ -0,0 +1,63 @@
+# Install
+
+Welcome to Widgetbook! This guide will help you get started with integrating Widgetbook into your Flutter projects and streamline your widget management process.
+
+Make sure to depend on the Widgetbook packages.
+
+There are two approaches to configure Widgetbook:
+
+1- **Manual Approach**: This approach involves using the [widgetbook package](https://pub.dev/packages/widgetbook) and manually maintaining the widget structure in Widgetbook to keep it in sync with your application.
+
+2- **Generator Approach**: This approach utilizes the [widgetbook_generator package](https://pub.dev/packages/widgetbook_generator) in combination with the [widgetbook_annotation package](https://pub.dev/packages/widgetbook_annotation). It automatically configures the widget structure in Widgetbook based on the structure defined in your application, keeping Widgetbook in sync with your app.
+
+<Tabs
+  groupId="approach"
+  values={[
+    { label: 'Generator', value: 'generator' },
+    { label: 'Manual', value: 'manual' },
+  ]}
+>
+  <TabItem value="generator">
+
+    ```bash
+    flutter pub add widgetbook_annotation widgetbook
+    flutter pub add widgetbook_generator build_runner --dev
+    ```
+
+    This will add a line like this to your package's pubspec.yaml (and run an implicit flutter pub get):
+
+```yaml
+dependencies:
+  widgetbook_annotation: ^3.0.0-beta.7
+
+dev_dependencies:
+  widgetbook: ^3.0.0-beta.14
+  widgetbook_generator: ^3.0.0-beta.11
+  build_runner:
+```
+
+  </TabItem>
+  <TabItem value="manual">
+
+    ```bash
+    flutter pub add widgetbook --dev
+    ```
+
+This will add a line like this to your package's pubspec.yaml (and run an implicit flutter pub get):
+
+```yaml
+dev_dependencies:
+  widgetbook: ^3.0.0-beta.14
+```
+
+  </TabItem>
+</Tabs>
+
+<Info>
+  We recommend using the Generator approach over the Manual approach, as it simplifies the
+  maintenance of Widgetbook. Additionally, the Generator approach offers seamless integration with
+  [Widgetbook Cloud](/widgetbook-cloud/overview), further enhancing the review and collaboration
+  process.
+</Info>
```

Comment:
I think this info should be before the TabView.

Reply:
fixed